src/main.rs

```diff
@@ -98,8 +98,12 @@ fn main() -> Result<(), Box<dyn Error>> {
         }
     };
     let path = matches.opt_str("output");
-    let commits_path = &matches.free[1];
-    let commits = fs::read_to_string(commits_path).unwrap();
+    let commits = std::process::Command::new("git")
+        .arg("log")
+        .arg("--numstat")
+        .output()
+        .expect("This is not a Git repository");
+    let commits = String::from_utf8(commits.stdout).unwrap();
```

Comment:
Okay, I am going to change the `expect` and `unwrap` to the suggested `?`.  Their usage originates from the examples in the official documentation.

Reply:
If you say "input as an option," how about making it literally an option?  Just like `-o`?

Reply:
> Okay, I am going to change the `expect` and `unwrap` to the suggested `?`. Their usage originates from the examples in the official documentation.

Yeah, especially in the beginning they are very helpful, to sketch out programs, without having to think about every possible error. My other rust programs also have a few (or maybe even a lot.. 😅 ) of them. But since this is rather small, and easily checked, something like the `?` operator is preferable. (Even though I'm not too sure, how I feel about it. Its output to the console is not the nicest imho)

> If you say "input as an option," how about making it literally an option? Just like `-o`?

Yes, I would be down for it. 😄

Reply:
:wrench: :heavy_check_mark: